modeldb/modelrun.py

```diff
@@ -209,16 +209,18 @@ def prepare_model(model):
             os.path.join(MODELS_ZIP_DIR, str(model.id) + ".zip"), "r"
     ) as zip_ref:
         model_dir = os.path.join(
-            model.working_dir, os.path.dirname(zip_ref.infolist()[0].filename)
+            model.working_dir,
+            str(model.id),
```

Comment:
Without this change then models from the ModelDB website (as opposed to those downloaded from GitHub) are extracted into `{working_dir}/{model_name}` where `model_name` is the directory name embedded in the zip file. These names are not unique, so some models overwrite each other.

Reply:
Will be dropped in favor of #72 (getting there)